avogadro/qtplugins/label/label.cpp

```diff
@@ -228,19 +303,32 @@ void Label::processAtom(const Core::Molecule& molecule,
       continue;
     }
 
-    auto text = atom.label();
-    if (text == "") {
-      text = Elements::symbol(atomicNumber) +
-             std::to_string(atomCount[atomicNumber]);
-    }
-    const Vector3f pos(atom.position3d().cast<float>());
     LayerLabel& interface = m_layerManager.getSetting<LayerLabel>(layer);
-    Vector3ub color = interface.color;
-    float radius = static_cast<float>(Elements::radiusVDW(atomicNumber)) *
-                   interface.radiusScalar;
-
-    TextLabel3D* atomLabel = createLabel(text, pos, radius, color);
-    geometry->addDrawable(atomLabel);
+    std::string text = "";
+    if (interface.atomOptions & LayerLabel::LabelOptions::Custom) {
+      text += (text == "" ? "" : " / ") + atom.label();
+    }
+    if (interface.atomOptions & LayerLabel::LabelOptions::Index) {
+      text += (text == "" ? "" : " / ") + std::to_string(atom.index());
```

Comment:
`atom.index() + 1` // most users expect Atom 1, Atom 2, etc. rather than starting at zero...

Reply:
Yes, but the "Select by Atom Index" is indexed at 0. I can change both elements to indexed at 1 if you prefer.

Reply:
Yes, I think most users expect atom numbers to start at 1.